src/components/course/data/hooks.jsx

```diff
@@ -576,9 +575,9 @@ export const useUserSubsidyApplicableToCourse = () => {
 
   const isSubscriptionLicenseApplicable = (
     subscriptionLicense?.status === LICENSE_STATUS.ACTIVATED
-    && catalogsWithCourse.includes(subscriptionPlan?.enterpriseCatalogUuid)
+    && subscriptionLicense?.subscriptionPlan.isCurrent
```

Comment:
[inform/question] FYI: do you think this may resolve https://2u-internal.atlassian.net/browse/ENT-8918, which was just injected into the new sprint during scrum?

I assigned the ticket to you, since I believe it might be the fix, or at least related? If not, we could rubber duck on what it might be! Otherwise, feel free to un-assign yourself as you see fit 😄

Reply:
Thanks for syncing this morning with me to validate that the current changes of the PR does resolve the use case of ticket 8918 👍🏽